From: Emmanuele Bassi <ebassi@gnome.org>
Date: Sun, 8 Jan 2023 00:28:43 +0000
Subject: Include GObject's visibility header

The gobject-introspection build goes through the GLib types when
generating the introspection data for GLib, but it does not include
glib-object.h, otherwise all GObject symbols would end up inside the
GLib namespace. This means we need to import the gobject-visibility.h
header inside glib-types.h. Since the header is guarded by a once
pragma, it doesn't really affect any legitimate user of the C API.

Origin: upstream, 2.75.3, commit:79fd62ccf26ea77ee3250276a5b2625db16758ce
---
 gobject/glib-types.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/gobject/glib-types.h b/gobject/glib-types.h
index 21081d2..87065b9 100644
--- a/gobject/glib-types.h
+++ b/gobject/glib-types.h
@@ -24,6 +24,7 @@
 #endif
 
 #include <glib.h>
+#include <gobject/gobject-visibility.h>
 
 G_BEGIN_DECLS
 
